pyls/__main__.py

```diff
@@ -95,3 +80,37 @@ def _binary_stdio():
         stdin, stdout = sys.stdin, sys.stdout
 
     return stdin, stdout
+
+def configure_logger(args):
+    log_config = args.log_config
+    root_logger = logging.root
+
+    if log_config:
+        with open(log_config, 'r') as f:
+            logging.config.dictConfig(json.load(f))
+    else:
+        log_file = args.log_file
+        formatter = logging.Formatter(LOG_FORMAT, style="%")
+        if log_file:
+            log_handler = logging.handlers.RotatingFileHandler(log_file, mode='a',
```

Comment:
Can you stick to repo parenthesis style? 
```
logging.handlers.RotatingFileHandler(
    log_file, mode='a', maxBytes=50*1024*1024, 
    backupCount=10, encoding=None, delay=0
)
```

Reply:
Oops.